### PR TITLE
Remove dependency on @syntest/cli for all packages

### PR DESCRIPTION
**Pull request by dstallenberg**

### CODE REVIEW

File: plugins/plugin-core-graphing/lib/GraphingPlugin.ts [hunk 1]
```diff
@@ -66,7 +65,7 @@ export default class GraphingPlugin extends ListenerPlugin {
       alias: [],
       default: "cfg",
       description: "The path where the csv should be saved",
-      group: CliOptionGroups.Storage,
+      group: "Graphing Options:",
```
Comment (mitchellolsthoorn) [inline plugins/plugin-core-graphing/lib/GraphingPlugin.ts:68]: Should we create an enum for this, even though we only have one instance for now?

File: libraries/module/lib/util/Configuration.ts [hunk 2]
```diff
@@ -16,25 +16,64 @@
  * limitations under the License.
  */
 import Yargs = require("yargs");
+import { LoggingOptions } from "@syntest/logging";
+
+export type GeneralOptions = {
```
Comment (mitchellolsthoorn) [inline libraries/module/lib/util/Configuration.ts:21]: The storage options make sense to move as the storage might change depending on the modules. However, it is a bit odd that the config type is in module but the parameter is in cli. We should maybe move the cli arg to the module as well. Does the cli have to define this?

Comment (dstallenberg) [reply libraries/module/lib/util/Configuration.ts:21]: Not necessarily. It just felt weird to have the usage of the tool defined in the module package. But I moved it now. The CLI now only contains the bin.ts file.

File: plugins/plugin-core-graphing/lib/GraphingPlugin.ts (hunk 1 above)
Comment (dstallenberg) [reply plugins/plugin-core-graphing/lib/GraphingPlugin.ts:68]: Added it.

**Review by mitchellolsthoorn — APPROVED** [review]

**Pull request merged**